galaxy/Dockerfile

```diff
@@ -125,6 +125,7 @@ RUN ansible-playbook /ansible/postgresql_provision.yml && \
     --extra-vars galaxy_extras_config_condor=True \
     --extra-vars galaxy_extras_config_condor_docker=True \
     --extra-vars galaxy_extras_config_rabbitmq=True \
+    --extra-vars galaxy_extras_config_cvmfs=True \
```

Comment:
Then again, not really because it should be using the default (https://github.com/galaxyproject/ansible-galaxy-extras/commit/56333c1f4ab54c5b39a60cc16fda9317f6e00c72). Was the latest cvmfs-tagged image (6ee8af117ebf) built using this default though? Timestamps would make one think yes so why is the variable not sticking...